lib/transmuxer.js

```diff
@@ -16,6 +16,9 @@ var TransportPacketStream, TransportParseStream, ElementaryStream,
     VideoSegmentStream, AudioSegmentStream, Transmuxer, AacStream,
     H264Stream, NalByteStream;
 
+// Helper functions
+var collectDTSInfo, clearDTSInfo, calculateTrackBMDT;
```

Comment:
DTS is a standard abbreviation so I think it's fine to use. I think "BMDT" is a bit more of a stretch and we should probably expand it.